js/arena.js

```diff
@@ -0,0 +1,38 @@
+import Gladiator from  "./gladiator.js";
+
+export default class Arena {
+    constructor(name){
+        this.name= name.toUpperCase()
```

Comment:
this will change the entire string to upper case we want only the first letter to be capitalised

away to do this is to use 

```
this.name.charAt(0).toUpperCase() + this._name.slice(1)
```